assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/vision/detection_predict.py

```diff
@@ -166,13 +167,22 @@ def predict(self, context: mlflow.pyfunc.PythonModelContext, input_data: pd.Data
         processed_images = input_data.loc[:, [MLflowSchemaLiterals.INPUT_COLUMN_IMAGE]].apply(
             axis=1, func=process_image_pandas_series
         )
-
+        if not params:
```

Comment:
nit: can be removed as params is already initialized to {} by default

Reply:
mlflow sends params=None when use calls predict without params, and in that case its failing when trying to access the keys